### PR TITLE
[risk=no][no ticket] Some small API code cleanups

### PR DESCRIPTION
**Pull request by jmthibault79**
No functional changes, just style/cleanup as I was passing thru

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/ProfileController.java [hunk 1]
```diff
@@ -17,7 +17,6 @@
 import javax.mail.MessagingException;
 import javax.mail.internet.AddressException;
 import javax.mail.internet.InternetAddress;
-import org.pmiops.workbench.access.AccessModuleService;
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/api/ProfileController.java:20]: these are not used

File: api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java [hunk 2]
```diff
@@ -26,8 +26,16 @@
 
   void deleteInstitution(final String shortName);
 
+  /**
+   * Update an institution in the DB if one matches the shortName.
+   *
+   * @param shortName
+   * @param updatedInstitution
+   * @return the updated institution, if an institution with the short name existed and the
+   *     operation was successful. Empty if there was no matching institution to update.
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java:35]: add a javadoc to make this behavior explicit

File: api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java [hunk 3]
```diff
@@ -440,12 +448,11 @@ private void setInstitutionEmailDomains(
       final List<DbAccessTier> dbAccessTiers) {
     institutionEmailDomainDao.deleteByInstitution(dbInstitution);
     for (DbAccessTier dbAccessTier : dbAccessTiers) {
-      institutionTierConfigMapper
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java:443]: these last 3 changes are the same

File: api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java [hunk 4]
```diff
@@ -174,28 +174,36 @@ public void deleteInstitution(final String shortName) {
   @Transactional
   @Override
   public Optional<Institution> updateInstitution(
-      final String shortName, final Institution institutionToUpdate) {
-    validateInstitution(institutionToUpdate);
+      final String shortName, final Institution updatedInstitution) {
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java:177]: same logic but a little clearer IMO

File: api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java [hunk 5]
```diff
@@ -174,28 +174,36 @@ public void deleteInstitution(final String shortName) {
...
-                    institutionDao.save(
-                        institutionMapper.modelToDb(institutionToUpdate).setInstitutionId(dbId));
-                populateAuxTables(institutionToUpdate, dbObjectToUpdate);
-                return toModel(dbObjectToUpdate);
-              });
+      institutionDao.save(dbObjectToUpdate);
+      populateAuxTables(updatedInstitution, dbObjectToUpdate);
     } catch (DataIntegrityViolationException ex) {
       throw new ConflictException(
-          "DataIntegrityException: Please check that you are not creating an Institute which already exists",
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java:196]: this text was incorrect

File: api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java [hunk 6]
```diff
@@ -174,28 +174,36 @@ public void deleteInstitution(final String shortName) {
...
+        .findOneByShortName(shortName)
+        .map(dbInst -> updateExistingInstitution(dbInst, updatedInstitution));
+  }
+
+  private Institution updateExistingInstitution(
+      DbInstitution dbInstitution, Institution updatedInstitution) {
+    // create new DB object, but mark it with the original's ID to indicate that this is
+    // an update
+    DbInstitution dbObjectToUpdate =
+        institutionMapper
```
Comment (NehaBroad) [inline api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java:191]: so neat and easy to read!!

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**